src/main/java/database/service/JsonDatabaseService.java

```diff
@@ -0,0 +1,272 @@
+package database.service;
+
+import database.exception.CreationDatabaseException;
+import database.exception.DatabaseDoesNotExistException;
+import database.exception.DeletionDatabaseException;
+import database.exception.DeserializeDatabaseException;
+import database.exception.IdDoesNotExistException;
+import database.exception.SerializeDatabaseException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import database.entity.BaseEntity;
+import database.helper.Settings;
+import database.helper.Validator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class JsonDatabaseService implements DatabaseService {
+    private final Settings settings;
+    private final ObjectMapper objectMapper;
+    private final Map<String, Integer> entityIds;
+    private static final Logger logger = LoggerFactory.getLogger(Settings.class);
+    public static final String EMPTY_BRACKETS_TO_JSON = "[]";
+
+    public JsonDatabaseService() throws CreationDatabaseException {
+        settings = new Settings();
+        objectMapper = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
+        entityIds = new HashMap<>();
+    }
+
+    public JsonDatabaseService(String propertyFileName) throws CreationDatabaseException {
+        settings = new Settings(propertyFileName);
+        objectMapper = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
+        entityIds = new HashMap<>();
+    }
+
+    @Override
+    public boolean createTable(Class<? extends BaseEntity> entityClass) {
+        File jsonDatabaseFile = new File(getDatabasePath(entityClass));
+        try {
+            if (!jsonDatabaseFile.createNewFile()) {
+                throw new CreationDatabaseException("Unable to create database file.");
+            }
+            return Files.writeString(jsonDatabaseFile.toPath(), EMPTY_BRACKETS_TO_JSON).toFile().exists();
+        } catch (IOException e) {
+            logger.error("Unable to create database file.");
+            throw new CreationDatabaseException("Unable to create database file.");
```

Comment:
Please create a variable with exception/log message if you use it multiple times. You can create a variable inside method instead of creating global constant.